cmd/gaia/testnets/README.md

```diff
@@ -137,15 +83,13 @@ make get_vendor_deps
 make install
 ```
 
-The next step is to copy the new genesis file:
+The next step is to copy the new genesis file.
 
 ```
 cp -a cmd/gaia/testnets/gaia-6001/genesis.json $HOME/.gaiad/config/genesis.json
 ```
 
-The last step is the adjust the `$HOME/.gaiad/config/config.toml`. Make sure that you are connected to healthy peers or seed nodes.
-These are some seeds nodes and they can be put into the config under the `seeds` key. Alternatively you can also
-ask user validators directly for a persistent peer and add it under the `persisent_peers` key.
+The last step is to update the `$HOME/.gaiad/config/config.toml`. Make sure that you are connected to healthy peers or seed nodes. These are some seeds nodes and they can be put into the config under the `seeds` key. You can also ask other validators for a persistent peer and add it under the `persisent_peers` key.
```

Comment:
I'd recommend adding "See [here](https://github.com/tendermint/tendermint/blob/develop/docs/using-tendermint.md#peers)" for more info on seed nodes and persistent peers. (Just to give people knowledge that more docs exist)

Reply:
I will add this!